### ISSUE TITLE
AG Grid - should not be setting fullWidthRowComponent twice

### ISSUE THREAD

**Issue by linkaiqiyi** [ag-grid/ag-grid#5562]
**I'm submitting a ...**  (check one with "x")
```
[*] bug report => see 'Providing a Reproducible Scenario'
[] feature request => do not use Github for feature requests, see 'Customers of AG Grid'
[] support request => see 'Requesting Community Support'
```

**Current behavior**

When using `domLayout: 'print'` and `:full-width-cell-renderer="'FullWidthCellRenderer'"` at the same time, the error `AG Grid - should not be setting fullWidthRowComponent twice` appears, and the rendering of the page becomes very laggy.
<img width="1440" alt="image" src="https://user-images.githubusercontent.com/87741822/190980865-0242d4ed-0ac9-4656-b2b0-6ec17065da8c.png">

**Expected behavior**

No error and smooth rendering of the tables.

**Please tell us about your environment:**

Google Chrome: 105.0.5195.102

* **AG Grid version:** V26.0.1

```
    "ag-grid-community": "^26.1.0",
    "ag-grid-enterprise": "^26.1.0",
    "ag-grid-vue": "^26.1.0"
```
* **Browser:**

'Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/105.0.0.0 Safari/537.36'

* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
ES6 | Vue2.0

**Comment by StephenCooper** [comment]
Can you please provide us with a reproduction so that we can investigate this. We will also like to validate if this is an issue in the latest version of the grid as I know that there has been work in this area of the grid between v26 and our current v28.

**Comment by linkaiqiyi** [comment]
Sorry for the error described above, not using `colSpan`, but `full-width-cell-renderer`, which has been modified.

**Comment by linkaiqiyi** [comment]
> [This is the codesandbox to reproduction.](https://codesandbox.io/s/ag-grid-5562-xgml1q)

The version of `Ag-Grid` is `v26.0.1`, but when I use `v27.0.1`, and I perform `work-break: break-word` on the content beyond the cell, then the table will keep beating.

**Comment by StephenCooper** [comment]
We believe that this issue is fixed in 27.3, so please upgrade to at least that version if you do not want to use the latest of v28
